### PR TITLE
[WD-10347] add missing word in section title at /download/server/arm

### PR DESCRIPTION
**Pull request by akbarkz**
## Done

- Add missing "and" word in one of the section's title

## QA

- Navigate to https://ubuntu-com-13741.demos.haus/download/server/arm
- Scroll down to second section and check its title if it's the same as specified in the [copy doc](https://docs.google.com/document/d/1whVU3DvB9j5k6Ed3UvLZCEN6BU-lZcow4ZefqKQqokg/edit#heading=h.aqoxcz2ugemk)
 
## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10347

## Screenshots

Screenshot 2024-04-12 at 10 47 17

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/download/server/arm.html [hunk 1]
```diff
@@ -64,16 +80,26 @@ <h2 class="p-heading--5">Ubuntu Server (64k page size)</h2>
   <hr class="p-rule is-fixed-width" />
   <div class="row--50-50">
     <div class="col">
-      <h2>Containers, databases, web <br class="u-hide--small u-hide--medium" /> more</h2>
+      <h2>
+        Containers, databases, web
+        <br class="u-hide--small u-hide--medium" />
+        and more
```
Comment (akbarkz) [inline templates/download/server/arm.html:86]: this is the actual fix, the rest is formatting by djLint

File: templates/download/server/arm.html [hunk 2]
```diff
@@ -121,10 +154,14 @@ <h3 class="p-heading--5">Installation guides</h3>
           <h3 class="p-heading--5">Professional support for Ubuntu</h3>
         </div>
         <div class="col-6 col-medium-3">
-          <p>Get professional support for Ubuntu from Canonical. We help organisations around the world to manage their
-            Ubuntu cloud, server and desktop deployments.</p>
+          <p>
+            Get professional support for Ubuntu from Canonical. We help organisations around the world to manage their
+            Ubuntu cloud, server and desktop deployments.
```
Comment (carkod) [inline templates/download/server/arm.html:159]: This break doesn't look right?

**Review by carkod — APPROVED** [review]

File: templates/download/server/arm.html (hunk 2 above)
Comment (carkod) [reply templates/download/server/arm.html:159]: Strange, I don't see the break in the conversation tab.

Comment (akbarkz) [reply templates/download/server/arm.html:159]: Don't see an issue here, on the page it shows correctly. I think linter applied 120 max symbols per line rule here.

**Pull request merged**